prac_05/state_names.py

```diff
@@ -4,4 +4,17 @@
 File needs reformatting
 """
 
+STATE_NAMES = {"QLD": "Queensland", "NSW": "New South Wales", "NT": "Northern Territory", "WA": "Western Australia",
```

Comment:
`STATE_NAMES` sounds like a (plural) list. Consider using the `x_to_y` naming convention for dictionaries.

Reply:
this one was named as a plural in the code to copy, BUT that's no excuse as i could have improved on existing code. So this is good feedback. Have updated now and will push tonight.